pmml_recommendation/kogito-pmml-recommender-client/src/main/java/org/kie/pmmlrecommendation/MainApp.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.pmmlrecommendation;
+
+import org.kie.pmmlrecommendation.dto.Customer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.stream.IntStream;
+
+import static org.kie.pmmlrecommendation.business.Recommender.getRecommendation;
+
+public class MainApp {
+
+    private static final Logger logger = LoggerFactory.getLogger(MainApp.class);
+
+    public static void main(String[] args) {
+        IntStream.range(0, 5).forEach(i -> {
+            Customer customer = new Customer();
+            logger.info("Customer {}", customer);
+            String recommendation = getRecommendation(customer);
+            logger.info("We recommend: {}", recommendation);
+        });
+    }
```

Comment:
Personally I don't like this main entry point: it is just a random generation of 5 customers with random data (moreover the random generation is hidden inside the empty constructor). I think that for demo purpose it is better to have a list of explicit/hardcoded customers in a method like `getCustomerData(customerId)` with a comment like `// this method mock the fetching from a DB/event of customer data to be classified`
Finally the `args` is the id/name of the Customer to be fetched, if empty it takes a default value.

So that user can easily start from there and replace this main class with some real logic if he wants to play with it.

Wdyt?

Reply:
@danielezonca 
Possibly. But this is how the original code is, and I like the idea to show to the user that to switch between environments only very few modifications are needed. Not strongly opinionated, though...